core/rawdb/accessors_chain.go

```diff
@@ -1190,17 +1195,26 @@ func ReadEtxSet(db ethdb.Reader, hash common.Hash, number uint64) types.EtxSet {
 	if len(data) == 0 {
 		return nil
 	}
-	etxSet := types.EtxSet{}
-	if err := rlp.Decode(bytes.NewReader(data), etxSet); err != nil {
+	var entries []etxSetEntry
```

Comment:
don't we need to pass in the reference for decoding?

Reply:
I pass a slice into `Decode()`, which is a reference. I believe this works as intended? But I can't test it until I have a test tool to emit ETXs and mine them in (coming soon ™️ )

Reply:
This was actually fixed in a later commit: 2c2530de6

I should squash them together 🤦

Reply:
I've squashed and force pushed